recipes/minhook/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "cci.20240114":
```

Comment:
We will be updating this versioning scheme, to be more future-proof. I'll do it myself and contribute.